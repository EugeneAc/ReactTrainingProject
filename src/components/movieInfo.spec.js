import React from 'react';
import renderer from 'react-test-renderer';

import MovieInfo from './movieInfo';

describe('Snapshot', () => {
it('MovieInfo matches the snapshot', () => {
    var movie = {
        imgSrc: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSJjJ0oj9DdOyVjYyDREytOES2pKEDgUtXDvNeaDnflAoXfg6I4",
                  movieName: "Kill Bill",
                  movieYear: "2003",
                  movieGenre: "Action",
                  rating:1,
                  movieDesc: "Kill Bill: Volume 1 is a 2003 American martial arts film written and directed by Quentin Tarantino. It stars Uma Thurman as the Bride, who swears revenge on a team of assassins (Lucy Liu, Michael Madsen, Daryl Hannah, and Vivica A. Fox) and their leader Bill (David Carradine) after they try to kill her and her unborn child. Her journey takes her to Japan, where she battles the Tokyo yakuza."
    }
    const tree = renderer.create(<MovieInfo movie={movie}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});