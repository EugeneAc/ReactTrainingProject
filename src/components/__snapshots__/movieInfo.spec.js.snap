// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot MovieInfo matches the snapshot 1`] = `
<div
  className="movie-info"
>
  <img
    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSJjJ0oj9DdOyVjYyDREytOES2pKEDgUtXDvNeaDnflAoXfg6I4"
  />
  <div
    className="movie-desc"
  >
    <div
      className="movie-name-info inline"
    >
      Kill Bill
    </div>
    <div
      className="movie-length-info inline"
    />
    <div
      className="movie-year-info inline"
    >
      2003
    </div>
    <div
      className="movie-decs-info inline"
    >
      Kill Bill: Volume 1 is a 2003 American martial arts film written and directed by Quentin Tarantino. It stars Uma Thurman as the Bride, who swears revenge on a team of assassins (Lucy Liu, Michael Madsen, Daryl Hannah, and Vivica A. Fox) and their leader Bill (David Carradine) after they try to kill her and her unborn child. Her journey takes her to Japan, where she battles the Tokyo yakuza.
    </div>
  </div>
</div>
`;
